Makefile

```diff
@@ -28,6 +29,10 @@ dev: python
 devdata:
 	@true
 
+.PHONY: shell
+shell: python
+	@tox -qe dev --run-command ipython
```

Comment:
This just runs IPython directly rather than Pyramid's pshell because [h-periodic is no longer going to include Pyramid](https://github.com/hypothesis/h-periodic/pull/49)